packages/manager/modules/pci/src/projects/project/kubernetes/components/anti-affinity/anti-affinity.html

```diff
@@ -0,0 +1,16 @@
+<oui-checkbox
+    data-model="$ctrl.nodePool.antiAffinity"
+    disabled="$ctrl.nodePool.nodeCount > 5"
```

Comment:
The `5` value is repeated quite a few times, a constant or exposing it through a method like `getMaxNodeCount()` would be better to maintain